[2025-03-28T18:12:02.648+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-28T18:12:02.803+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Etl_pipeline_order_management.fetch_data scheduled__2025-03-28T18:09:00+00:00 [queued]>
[2025-03-28T18:12:02.811+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Etl_pipeline_order_management.fetch_data scheduled__2025-03-28T18:09:00+00:00 [queued]>
[2025-03-28T18:12:02.812+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-28T18:12:02.826+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_data> on 2025-03-28 18:09:00+00:00
[2025-03-28T18:12:02.838+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2068) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-28T18:12:02.841+0000] {standard_task_runner.py:72} INFO - Started process 2070 to run task
[2025-03-28T18:12:02.840+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Etl_pipeline_order_management', 'fetch_data', 'scheduled__2025-03-28T18:09:00+00:00', '--job-id', '94', '--raw', '--subdir', 'DAGS_FOLDER/Gestioncommands.py', '--cfg-path', '/tmp/tmp5n0q07zw']
[2025-03-28T18:12:02.842+0000] {standard_task_runner.py:105} INFO - Job 94: Subtask fetch_data
[2025-03-28T18:12:02.913+0000] {task_command.py:467} INFO - Running <TaskInstance: Etl_pipeline_order_management.fetch_data scheduled__2025-03-28T18:09:00+00:00 [running]> on host 499696d11f67
[2025-03-28T18:12:03.005+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Etl_pipeline_order_management' AIRFLOW_CTX_TASK_ID='fetch_data' AIRFLOW_CTX_EXECUTION_DATE='2025-03-28T18:09:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-28T18:09:00+00:00'
[2025-03-28T18:12:03.008+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-03-28T18:12:03.009+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-03-28T18:12:03.009+0000] {logging_mixin.py:190} INFO - Current task name:fetch_data state:running start_date:2025-03-28 18:12:02.804117+00:00
[2025-03-28T18:12:03.010+0000] {logging_mixin.py:190} INFO - Dag name:Etl_pipeline_order_management and current dag run status:running
[2025-03-28T18:12:03.010+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-28T18:12:03.011+0000] {Gestioncommands.py:28} INFO - Fetching data from API...
[2025-03-28T18:12:04.673+0000] {Gestioncommands.py:32} INFO - Data fetched from API successfully.
[2025-03-28T18:12:04.730+0000] {Gestioncommands.py:36} INFO - CSV data pushed to XCom.
[2025-03-28T18:12:04.732+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-28T18:12:04.743+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-28T18:12:04.743+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=Etl_pipeline_order_management, task_id=fetch_data, run_id=scheduled__2025-03-28T18:09:00+00:00, execution_date=20250328T180900, start_date=20250328T181202, end_date=20250328T181204
[2025-03-28T18:12:04.769+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-03-28T18:12:04.770+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-03-28T18:12:04.771+0000] {logging_mixin.py:190} INFO - Dag name:Etl_pipeline_order_management queued_at:2025-03-28 18:12:00.477448+00:00
[2025-03-28T18:12:04.771+0000] {logging_mixin.py:190} INFO - Task hostname:499696d11f67 operator:PythonOperator
[2025-03-28T18:12:04.792+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-28T18:12:04.803+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
